app/instrumentation-tests/src/org/commcare/androidTests/LazyVideosTests.kt

```diff
@@ -114,6 +117,15 @@ class LazyVideosTests : BaseTest() {
                 R.drawable.icon_info_outline_lightcool
             )
         ).perform(click())
+        if(onView(withTagValue(CoreMatchers.`is`(R.drawable.update_download_icon))).isPresent()){
```

Comment:
why do we have a `if` here ?

Reply:
Same reason as the one before. Sometimes the videos for the other questions may not get downloaded properly.